src/renderer/components/editorWithTabs/editor.vue

```diff
@@ -235,6 +236,12 @@ export default {
         }
       }
     },
+    sequenceTheme: function (value, oldValue) {
+      const { editor } = this
+      if (value !== oldValue && editor) {
+        editor.setOptions({ sequenceTheme: value })
```

Comment:
I prefer use

```js
editor.setOptions({ sequenceTheme: value }, true)
```

This will force re-render the editor.

Reply:
Perfect! I was looking for an option like that!